### PR TITLE
Freeland Lesson 03 Assignment

### PR DESCRIPTION
**Pull request by ryanfree1994**
I am submitting my lesson 03 assignment.

### CODE REVIEW

File: lesson03/assignment/css/style.scss [hunk 1]
```diff
@@ -3,4 +3,105 @@
...
+  &__first-background{
+    display: $flex;
+    flex-direction: $horizontal;
+    flex-wrap: $wrap;
+    padding: 1em;
+    background-color: $white;
+    width: 52.938em;
+    height: 23.938em;
+
+  //media query mixin to change flex direction to columns and height and width for screen//
```
Comment (tylervanbrocklin) [inline lesson03/assignment/css/style.scss:32]: great comments throughout, makes reading your code very nice and clean.

File: lesson03/assignment/scripts/script.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    "email": "mJordan@bulls.com",
+    "linkedInUrl": "mJordas.linkedinprofile.com",
+    "codeLanguages": [
+      "HTML", "CSS", "JavaScript", "Java", "Spring"
+    ]
+  }
+];
+
+/*creating template function*/
+const linkedInHTML = linkedIn_data.map(function(template) {
```
Comment (tylervanbrocklin) [inline lesson03/assignment/scripts/script.js:57]: your JSON data is constructed well, and you're mapping the data correctly. Nothing to add here!

File: lesson03/assignment/scripts/script.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+          <p class="main__p-content"><b>Languages: </b>${template.codeLanguages[0]}, ${template.codeLanguages[1]}, ${template.codeLanguages[2]}, ${template.codeLanguages[3]}, ${template.codeLanguages[4]}</p>
+        </div>
+      </div>
+  </div>`;
+  return linkedIn_person;
+});
+
+/*inserting templates based on length of JSON data*/
+const insert_template = document.getElementById("template-hook");
+
```
Comment (tylervanbrocklin) [inline lesson03/assignment/scripts/script.js:83]: You might find it easier to have a wrapping div in your html, then use jQuery to append your cards inside of it. $(".template-hook").append(linkedInHTML);

Comment (ryanfree1994) [reply lesson03/assignment/scripts/script.js:83]: I tried this earlier in the assignment, yet I continue to receive the following error: "$ is not defined."

**Pull request merged**